SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -176,7 +181,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = SWIFT_PACKAGE;
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Default;
+				SWIFT_VERSION = 4.0;
```

Comment:
These changes look good otherwise, but will the project still build on current Xcode if you set SWIFT_VERSION to 4.0? I would prefer leaving it at 3.0 for now.